perfs_patch_arrangement.rst

```diff
@@ -0,0 +1,116 @@
+Patch arrangement
+=================================
+
+For some case such as very local plasmas, the dynamic load balancing could reach limits :
+ * it needs computational load to balance
+ * it fits dynamic problems
+
+For weak particles dynamic it could be preferable to masterize the patch distribution over MPI.
```

Comment:
What does masterize mean???

Reply:
"... regarding an arrangement along an Hilbert curve" ?

Reply:
I still don't understand what you want to say at all. I guess I am missing an important point, but this sentence is strange to me.

Reply:
Maybe it is not the good word, I'll illustrate the idea below. And I let you reformulate it.  
The idea is that, for instance, using an hilbertian arrangement for a thin solid target is not intuitive at all to estimate the number of processes which will really run particles.  
The linearized arrangement can.